crates/holochain/Cargo.toml

```diff
@@ -317,6 +317,9 @@ chc = ["bytes", "reqwest", "holochain_conductor_api/chc", "holochain_chc/http"]
 # Transitional feature flag for code that is only ready when DPKI integration lands.
 dpki = []
 
+# Enable sharding for networks > 100 nodes
+network-sharding = []
```

Comment:
This should probably go into the default CI features and then we'll look at testing without those separately